themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md

```diff
@@ -3,13 +3,11 @@
 title: "Introduction to Infrastructure as Code with Pulumi YAML"
```

Comment:
title: Workshop: Introduction to Infrastructure as Code with Pulumi YAML

Reply:
@shughes26 can you get this title updated to be what Sara is describing above?

Reply:
https://github.com/pulumi/pulumi-hugo/pull/1739 created a new PR with the change

Reply:
Looks like the updated title exceeds the title character limit by 5 characters (60 characters max)